internal/cmd/cmd.go

```diff
@@ -73,8 +73,8 @@ func printTable(data interface{}, out io.Writer) {
 	table.Print(data)
 }
 
-// Creates a new API Client from the current config
-func defaultAPIClient() (*api.Client, error) {
+// Creates a new API Client from the current config and command flags
+func defaultAPIClient(flags *pflag.FlagSet) (*api.Client, error) {
```

Comment:
This feels heavy handed for a single flag. It also precludes using environment variables to set `ignore-version`

Reply:
I did it this way because we would have to parse the flag everywhere otherwise. Added the environment variable though.